pkg/sdk/go/keys.go

```diff
@@ -6,43 +6,58 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"time"
 
 	"github.com/mainflux/mainflux/pkg/errors"
 )
 
+type keyReq struct {
+	Type     uint32        `json:"type,omitempty"`
+	Duration time.Duration `json:"duration,omitempty"`
+}
+
 const keysEndpoint = "keys"
+const (
+	// LoginKey is temporary User key received on successfull login.
+	LoginKey uint32 = iota
+	// RecoveryKey represents a key for resseting password.
+	RecoveryKey
+	// APIKey enables the one to act on behalf of the user.
+	APIKey
+)
```

Comment:
Can you use these constants from Auth service? Please check if that's OK from dependencies perspective (we don't want to pull all the Auth service and its dependencies), but I think that it should be.